dev-python/verboselogs/verboselogs-1.7.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Verbose logging for Python's logging module"
+HOMEPAGE="https://pypi.org/project/verboselogs/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="MIT"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
```

Comment:
I see the upstream pypi tarball has docs/ directory - do they get installed by default? Maybe a candidate for `docs` IUSE?